tools/simple_test.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# ------------------------------------------------------------------------------
+# This is meant to be a somewhat self contained script for quickly
+# proving an Ethereum mainnet block with the type 1 prover. The goal
+# is to use this for benchmarking and CI. This is the block in
+# question: https://etherscan.io/block/19240705
+
+# We're going to set the paralellism in line with the total cpu count
+num_procs=$(nproc)
+
+2>&1 echo "Pulling sample witness"
+witness_cid_hash="QmbwnLGuZ2qxZDqETAFb5DnyjZry8Sv3UFwYnsgKmsE3of"
+curl -s -L "https://cf-ipfs.com/ipfs/$witness_cid_hash" > witness.json.bz2
+bunzip2 -f witness.json.bz2
+
+# Configured Rayon and Tokio with rough defaults
+export RAYON_NUM_THREADS=$num_procs
+export TOKIO_WORKER_THREADS=$num_procs
+
+export RUST_MIN_STACK=33554432
+export RUST_BACKTRACE=full
+export RUST_LOG=info
+export RUSTFLAGS='-C target-cpu=native'
+
+# This sizes are configured specifically for this witness. Don't use this in other scenarios
+export ARITHMETIC_CIRCUIT_SIZE="15..19"
+export BYTE_PACKING_CIRCUIT_SIZE="9..19"
+export CPU_CIRCUIT_SIZE="12..21"
+export KECCAK_CIRCUIT_SIZE="14..18"
+export KECCAK_SPONGE_CIRCUIT_SIZE="9..13"
+export LOGIC_CIRCUIT_SIZE="12..17"
+export MEMORY_CIRCUIT_SIZE="17..23"
```

Comment:
My concern with this, though not really a concern per say, is that having it in `tools/` may get people to use it, which will flush all their previous circuits and incur kind of unnecessary overhead.

One option would be to have instead a `test_witnesses` folder including this and possibly other blocks, and keep the script itself in the `.ci/` folder? If one wanted to use this witness, they could then easily do it with their own config by using the `stdio` mode.

Reply:
Also nitpicky comment, but `test-only` doesn't require circuits, so we could conditionally set the ranges to minimal sizes, similarly to `debug_block.sh`

Reply:
> E.g. I think there is a case that someone will want to quickly benchmark a system and they can just run this script two or three times in a row to get a sense of the performance of their system.

Makes sense. Let's maybe keep it here as you suggest then.

> Regarding the test only feature - my instinct would be to delete that option from this particular script and just have a different one for in the zk_evm repo.

Removing it would focus on the full e2e testing / benchmarking approach, which I'm not against (as again, regressions should ideally be caught before even making it here). As for the sibling test on the `zk_evm` repo, we could just have it as integration test of the decoder I think.